cmd/ttn-lw-cli/commands/use.go

```diff
@@ -0,0 +1,140 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/tls"
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"go.thethings.network/lorawan-stack/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+func destPath(base string, user bool, overwrite bool) (string, error) {
+	fileName := base
+	if user {
+		dir, err := os.UserConfigDir()
+		if err != nil {
+			return "", err
+		}
+		fileName = filepath.Join(dir, base)
+	}
+
+	_, err := os.Stat(fileName)
+	if err == nil && !overwrite {
+		logger.Warnf("%s already exists. Use --overwrite", fileName)
+		return "", errFileExists.WithAttributes("file", fileName)
+	}
+
+	return fileName, nil
+}
+
+var (
+	errNoHost      = errors.DefineInvalidArgument("no_host", "no host set")
+	errFileExists  = errors.DefineAlreadyExists("file_exists", "`{file}` exists")
+	errFailWrite   = errors.DefinePermissionDenied("fail_write", "failed to write `{file}`")
+	configFileName = ".ttn-lw-cli.yml"
+	caFileName     = "ca.pem"
+
+	useCommand = &cobra.Command{
+		Use:               "use [host]",
+		Short:             "Use",
+		PersistentPreRunE: preRun(),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			insecure, _ := cmd.Flags().GetBool("insecure")
+			ca, _ := cmd.Flags().GetBool("ca")
+			user, _ := cmd.Flags().GetBool("user")
+			overwrite, _ := cmd.Flags().GetBool("overwrite")
+			credentialsID, _ := cmd.Flags().GetString("credentials-id")
+
+			if len(args) != 1 {
+				return errNoHost
+			}
+
+			host := args[0]
+			conf := MakeDefaultConfig(host, insecure)
+			if credentialsID != "" {
+				conf.CredentialsID = credentialsID
+			}
+
+			// Get CA certificate from server
+			if !insecure && ca {
+				logger.Infof("Will retrieve certificate from %s", conf.NetworkServerGRPCAddress)
+				caFile, err := destPath("ca.pem", user, overwrite)
+				if err != nil {
+					return err
+				}
+
+				conn, err := tls.Dial("tcp", conf.NetworkServerGRPCAddress, &tls.Config{InsecureSkipVerify: true})
+				if err != nil {
+					return err
+				}
+				defer conn.Close()
+
+				var b bytes.Buffer
```

Comment:
```suggestion
				var buf bytes.Buffer
```
`b` is usually used for `[]byte`